bin/fetch-cpan.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
```

Comment:
Would be nice if all scripts could at least be called safely with `--help`/`-h`.

Reply:
Ok, but this one is just a wrapper for convenience and has no options at all.

Reply:
Doesn't that make it even more dangerous? I mean, it's in the same folder as the other scripts. I'd probably try running it with `--help`.

Reply:
ok

Reply:
done